src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,48 @@
+.watch {
+  @include circle($widthClock);
+
+  border: 1vmin dotted $colorBorder;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &::before {
+    @include after-before($widthSecond, $heightSecond, $colorSecond, 2);
+
+    animation-duration: $stepSecond;
+    animation-timing-function: linear;
+  }
```

Comment:
This block of code is a bit confusing. It's not clear what the `::before` selector is doing here. I suggest adding a comment to explain its purpose.